src/components/Books.css

```diff
@@ -0,0 +1,57 @@
+.book_container {
+  gap: 1rem;
+  margin-bottom: 3rem;
+  border-radius: 4px;
+  border: solid 1px #e8e8e8;
+  background-color: #fff;
+  padding: 1rem;
+  justify-content: space-between;
+}
+
+.book_status {
+  justify-content: space-between;
+  gap: 5rem;
+  align-self: auto;
+}
+
+.book {
+  flex-direction: column;
```

Comment:
- [ ] Please, consider fixing some issues in your styling, to be closer to the original Zeplit design:

1) Please, consider the margin between the card, as I showed in this screenshot below. The spaces between cards should be not so much, considering the original design 👍 

2) Please, consider aligning the text left in this section I marked 2, on the screenshot below.

> your design
<img width="1293" alt="Screen Shot 2022-08-19 at 01 06 08" src="https://user-images.githubusercontent.com/49064106/185496126-32cea6a5-8e2e-4194-83a0-5bb036390955.png">

> original one
<img width="887" alt="Screen Shot 2022-08-19 at 01 14 02" src="https://user-images.githubusercontent.com/49064106/185496239-c81a2921-93b9-4c25-bf7a-139edc5901d9.png">